en/docs/assets/css/blue-palette-alt1.css

```
@@ -0,0 +1,104 @@
+
+:root {
+    --md-primary-fg-color: #000000;
+    --md-primary-fg-color--light: #ffffff;
+    --md-primary-fg-color--dark: #147ecc;
+    --md-primary-bg-color: #fff;
+    --md-primary-bg-color--light: #ffffffb3;
+    --md-accent-fg-color: #033982;
+    --md-accent-fg-color--transparent: #0339821a;
+    --md-accent-bg-color: #fff;
+    --md-accent-bg-color--light: #ffffffb3;
+    --md-banner: url(../img/home/bi.webp) no-repeat right top;
```

Stylelint (17.4.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)